Formula/open-mpi.rb

```diff
@@ -5,6 +5,31 @@ class OpenMpi < Formula
   sha256 "663450d1ee7838b03644507e8a76edfb1fba23e601e9e0b5b2a738e54acd785d"
   revision 1
 
+  stable do
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do `patch do` blocks outside of any `SoftwareSpec` block apply the patches they pull in to the `stable` spec?  I couldn't remember if that was the case off of the top of my head…